### PR TITLE
added detail to the note section

### PR DESCRIPTION
**Pull request by MaratMussabekov**
source: https://docs.microsoft.com/en-us/windows/client-management/determine-appropriate-page-file-size
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9392

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/client-management/introduction-page-file.md [hunk 1]
```diff
@@ -67,4 +67,4 @@ The system commit charge is the total committed or "promised" memory of all comm
 The system committed charge and system committed limit can be measured on the **Performance** tab in Task Manager or by using the "\Memory\Committed Bytes" and "\Memory\Commit Limit" performance counters. The \Memory\% Committed Bytes In Use counter is a ratio of \Memory\Committed Bytes to \Memory\Commit Limit values.
 
 >[!Note]
```
Comment (JohanFreelancer9) [inline windows/client-management/introduction-page-file.md:69]:
```suggestion
>[!NOTE]
```

File: windows/client-management/introduction-page-file.md [hunk 2]
```diff
@@ -67,4 +67,4 @@ The system commit charge is the total committed or "promised" memory of all comm
...
->System-managed page files automatically grow up to three times the physical memory or 4 GB (whichever is larger) when the system commit charge reaches 90 percent of the system commit limit. This assumes that enough free disk space is available to accommodate the growth.
+>System-managed page files automatically grow up to three times the physical memory or 4 GB (whichever is larger, but no more than 1/8 of the volume size) when the system commit charge reaches 90 percent of the system commit limit. This assumes that enough free disk space is available to accommodate the growth.
```
Comment (JohanFreelancer9) [inline windows/client-management/introduction-page-file.md:70]:
```suggestion
>System-managed page files automatically grow up to three times the physical memory or 4 GB (whichever is larger, but no more than one-eighth of the volume size) when the system commit charge reaches 90 percent of the system commit limit. This assumes that enough free disk space is available to accommodate the growth.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review,

File: windows/client-management/introduction-page-file.md (hunk 1 above)
Comment (illfated) [reply windows/client-management/introduction-page-file.md:69]: If possible, please also add a space after the MD indent marker (`>`).

File: windows/client-management/introduction-page-file.md (hunk 2 above)
Comment (illfated) [reply windows/client-management/introduction-page-file.md:70]: If possible, please also add a space after the MD indent marker (`>`).

**Review by illfated — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Pull request merged**